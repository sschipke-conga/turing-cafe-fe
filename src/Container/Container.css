.container {
display: flex;
flex-wrap: wrap;
justify-content: flex-start;
margin-top: 20px;
width: 95%;
}