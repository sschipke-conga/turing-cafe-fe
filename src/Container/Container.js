import React from 'react';
import Reservation from '../Reservation/Reservation'

const Container = ({reservations}) => {
  let allReservations = reservations.map(reservation => {
    return <Reservation key={reservation.id} reservation={reservation} />
  })
  return (<section className="container">
    {allReservations}
  </section>)
}

export default Container