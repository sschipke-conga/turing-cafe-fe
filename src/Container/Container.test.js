import React from 'react';
import { shallow } from 'enzyme';
import Container from './Container'

describe('Container', () => {
  let wrapper, mockReservations;
  const mockCancelReservation = jest.fn();
  beforeEach(() => {
    mockReservations = [{
      id: 123,
      name: 'Kimberly',
      date: '10/25',
      time: '8:00',
      number: 6
    }]
    wrapper = shallow(
      <Container reservations={mockReservations} cancelReservation={mockCancelReservation} />
    )
  })
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})