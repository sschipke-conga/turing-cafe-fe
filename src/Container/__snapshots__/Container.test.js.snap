// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should match the snapshot 1`] = `
<section
  className="container"
>
  <Reservation
    cancelReservation={[MockFunction]}
    key="123"
    reservation={
      Object {
        "date": "10/25",
        "id": 123,
        "name": "Kimberly",
        "number": 6,
        "time": "8:00",
      }
    }
  />
</section>
`;
